src/heitech.zer0mqXt.core/Main/Zer0Mq.cs

```diff
@@ -28,6 +28,14 @@ public IZer0MqBuilder SetSerializer(ISerializerAdapter adapter)
             return this;
         }
 
+        public IZer0MqBuilder UseNewtonsoftJson()
```

Comment:
All tests are now using the new `System.Text.Json` serialization per default.
Perhaps adding one test which confirms that `Newtonsoft.Json` continues to work as expected would be nice.

Reply:
totally agree. Will do

Reply:
done